src/analysis/complex/locally_uniform_limit.lean

```diff
@@ -0,0 +1,167 @@
+/-
+Copyright (c) 2022 Vincent Beffara. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Vincent Beffara
+-/
+import analysis.complex.removable_singularity
+import analysis.calculus.uniform_limits_deriv
+
+/-!
+# Locally uniform limits of holomorphic functions
+
+## Main results
+
+* `tendsto_locally_uniformly_on.differentiable_on`: A locally uniform limit of holomorphic functions
+  is holomorphic.
+* `tendsto_locally_uniformly_on.deriv`: Locally uniform convergence implies locally uniform
+  convergence of the derivatives to the derivative of the limit.
+-/
+
+open set metric measure_theory filter complex interval_integral
+open_locale real topological_space
+
+variables {E ι : Type*} [normed_add_comm_group E] [normed_space ℂ E] [complete_space E]
+  {U K : set ℂ} {z : ℂ} {M r δ : ℝ} {φ : filter ι} {F : ι → ℂ → E} {f g : ℂ → E}
+
+namespace complex
+
+section cderiv
+
+/-- A circle integral which coincides with `deriv f z` whenever one can apply the Cauchy formula for
+the derivative. It is useful in the proof of the Weierstrass theorem because it depends continuously
+on `f` for the uniform topology. -/
+noncomputable def cderiv (r : ℝ) (f : ℂ → E) (z : ℂ) : E :=
+  (2 * π * I : ℂ)⁻¹ • ∮ w in C(z, r), ((w - z) ^ 2)⁻¹ • f w
+
+lemma cderiv_eq_deriv (hU : is_open U) (hf : differentiable_on ℂ f U) (hr : 0 < r)
+  (hzr : closed_ball z r ⊆ U) :
+  cderiv r f z = deriv f z :=
+two_pi_I_inv_smul_circle_integral_sub_inv_smul_of_differentiable hU hzr hf (mem_ball_self hr)
```

Comment:
The name `two_pi_I_inv_smul_circle_integral_sub_inv_smul_of_differentiable` is really strange here, because it looks like the name for the usual Cauchy formula, not for the derivative. Could you fix the name, by putting in `sq` somewhere?